### PR TITLE
Refactor/search bar error message

### PR DESCRIPTION
**Pull request by jmyacobn**
Summary of the work done:
- Modified "Search..." to "Search by recipe name" in Search Bar placeholder
- Added a conditional to searchHomeRecipeByName and searchFavoriteRecipeByName functions that gives error message if user does not put anything in Search bar OR the user input does not match a stored recipe

This branch includes:

- Changes to index.html and scripts.js

Contributed to code:

- Jenn

Needs to review:

- Eleanor and Kiko

Additional notes about any issues/questions remaining around this code:

- I did not reformat in CSS, but I am willing to do this down the road if you think it is necessary!

Indicate anything that still needs to be addressed

- N/A

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -20,7 +20,7 @@ <h2 class="user-info" id="user-info">Welcome, User!</h2>
     <main class="main">
       <section class="filter-section" id="filterSection">
         <form class="search">
-            <input type="text" placeholder="Search..." id="search-bar">
+            <input type="text" placeholder="Search by recipe name" id="search-bar">
```
Comment (knpausch) [inline dist/index.html:23]: I like this improvement, it gives directions to the user passively

**Review by knpausch — APPROVED** [review]
Great work Jen! I ran your branch and it worked beautifully

**Comment by knpausch** [comment]
Had to close this PR. Lauren's Refactor: Radio Button is going to counter my code once it merges. I'll have to pull from the new main and update my changes

**Comment by knpausch** [comment]
Omg sorry, I closed the wrong PR haha. I re-opened this. Please disregard

File: dist/index.html (hunk 1 above)
Comment (Eleanorgruth) [reply dist/index.html:23]: I agree! I really like this, great job Jenn!

File: src/scripts.js [hunk 2]
```diff
@@ -210,6 +213,9 @@ function searchFavoriteRecipeByName() {
     filteredFavorites.forEach((current) => {
         displayRecipePreview(current, favoritesView)
     })
+    if(filteredFavorites.length === 0 || searchBar.value === '') {
+       favoritesView.innerHTML = `<p>No recipes found. Please search by recipe name, or select a category to filter recipes.</p>`
```
Comment (Eleanorgruth) [inline src/scripts.js:217]: This error message also looks great!

Comment (Eleanorgruth) [reply src/scripts.js:217]: It looks like this code is repetitive though, could we make it a helper function and then have a parameter to pass in the view?

**Review by Eleanorgruth — COMMENTED** [review]
I think that we should make that error function a helper function to DRY up the code, its up to you if you want to do that now or later?

**Pull request merged**